### PR TITLE
Card navbar created

### PR DESCRIPTION
**Pull request by yashodaAgarwal**
Please review my card and navbar component.

### CODE REVIEW

File: style/Card.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  }
+  img {
+    height: 200px;
+    width: 200px;
+    margin-bottom: 0;
+    padding: 0;
+  }
+  .card-container {
+    width: 200px;
+    height: 300px;
```
Comment (theVivek-kumar) [inline style/Card.css]: use rem

File: style/Card.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    color: var(--white-color);
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+  }
+
+  .image-overlay {
+    position: absolute;
+    top: 0;
+    width: 200px;
+    height: 200px;
+    background-color: rgba(0, 0, 0, 0.6);
```
Comment (theVivek-kumar) [inline style/Card.css]: use color variable

File: style/Card.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  }
+  
+  .image-overlay:hover {
+    opacity: 1;
+  }
+
+  .text-card-container {
+    width: 250px;
+    height: 180px;
+    background-color: var(--white-color);
```
Comment (theVivek-kumar) [inline style/Card.css:64]: like this this is good

File: style/navigation.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
+.navigation-item{
+    padding:20px;
```
Comment (theVivek-kumar) [inline style/navigation.css]: above 10px use rem

File: style/navigation.css [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    align-items: center;
+    justify-content: space-around;
+    width:300px;    
+}
+.nav-heading{
+    font-size:20px;
+    text-decoration: none;
+    color:var(--black-color);
+    font-weight: 800;
+    padding-left: 60px;
```
Comment (theVivek-kumar) [inline style/navigation.css]: rem

File: style/Card.css [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    flex-direction: column;
+    opacity: 0;
+  }
+  
+  .image-overlay:hover {
+    opacity: 1;
+  }
+
+  .text-card-container {
+    width: 250px;
```
Comment (theVivek-kumar) [inline style/Card.css]: image size in %

**Review by theVivek-kumar — CHANGES_REQUESTED** [review]

File: component.html [hunk 7]
```diff
@@ -305,7 +305,156 @@ <h3 class="avatar-heading">Floating action Button</h3>
...
 
-    
+    <!-- --- Card ---  -->
+
+    <section class="main-title" id="cards">
+      <h2 class="avatar-heading">Cards</h2>
+      <p class="component-navbar-item">
+        Card are used to show user related data collectively, like product details.
+      </p>
+      <h3 class="avatar-heading">Cards with badges</h3>
```
Comment (SinchanaMS) [inline component.html]: change class name "avatar-heading"  to a generic name.

Comment (yashodaAgarwal) [reply component.html]: done

File: style/Card.css (hunk 1 above)
Comment (yashodaAgarwal) [reply style/Card.css]: done

File: style/Card.css (hunk 2 above)
Comment (yashodaAgarwal) [reply style/Card.css]: done

File: style/navigation.css (hunk 4 above)
Comment (yashodaAgarwal) [reply style/navigation.css]: done

File: style/navigation.css (hunk 5 above)
Comment (yashodaAgarwal) [reply style/navigation.css]: okay

File: style/Card.css (hunk 6 above)
Comment (yashodaAgarwal) [reply style/Card.css]: okay

**Review by theVivek-kumar — COMMENTED** [review]
looks good now can be merged

**Pull request merged**